src/js/structs/Job.js

```diff
@@ -127,6 +128,6 @@ module.exports = class Job extends Item {
   }
 
   toJSON() {
-    return this.get();
+    return cleanJobJSON(this.get());
```

Comment:
We're going to refactor the service structs so that we have a `ServiceSpec` which will only include the "definition" and eventually `ServiceStatus` including all the status informations (the data removed by the clean methods). Thus we don't need those method any more.